test/machineslib.py

```diff
@@ -25,21 +25,21 @@
 class VirtualMachinesCaseHelpers:
     created_pool = False
 
-    def performAction(self, vmName, action, checkExpectedState=True):
+    def performAction(self, vmName, action, checkExpectedState=True, connectionName="system"):
         b = self.browser
-        b.click("#vm-{0}-action-kebab button".format(vmName))
-        b.wait_visible("#vm-{0}-action-kebab > .pf-c-dropdown__menu".format(vmName))
-        b.click("#vm-{0}-{1} a".format(vmName, action))
+        b.click("#vm-{0}-{1}-action-kebab button".format(vmName, connectionName))
+        b.wait_visible("#vm-{0}-{1}-action-kebab > .pf-c-dropdown__menu".format(vmName, connectionName))
+        b.click("#vm-{0}-{1}-{2} a".format(vmName, connectionName, action))
 
         if not checkExpectedState:
             return
 
         if action == "pause":
-            b.wait_in_text("#vm-{0}-state".format(vmName), "Paused")
+            b.wait_in_text("#vm-{0}-{1}-state".format(vmName, connectionName), "Paused")
         if action == "resume" or action == "run":
-            b.wait_in_text("#vm-{0}-state".format(vmName), "Running")
+            b.wait_in_text("#vm-{0}-{1}-state".format(vmName, connectionName), "Running")
         if action == "forceOff":
-            b.wait_in_text("#vm-{0}-state".format(vmName), "Shut off")
+            b.wait_in_text("#vm-{0}-{1}-state".format(vmName, connectionName), "Shut off")
```

Comment:
In the future we want to use f-strings everywhere I think, but as it was already there approved.